control/pzmap.py

```diff
@@ -40,11 +40,42 @@
 #
 # $Id:pzmap.py 819 2009-05-29 21:28:07Z murray $
 
-from numpy import real, imag
-from .lti import LTI
+from numpy import real, imag, linspace, pi, exp, cos, sin, sqrt
+from .lti import LTI, isdtime, isctime
 
 __all__ = ['pzmap']
 
+def zgrid(plt):
+    for zeta in linspace(0, 0.9,10):
```

Comment:
The range for zeta should probably be something with a default set of parameters that can be reset by the user.

Reply:
making `zetas` and `wns` arguments so user can override default behaviour.